cmd/xcode.go

```diff
@@ -67,7 +67,8 @@ the one you usually open in Xcode, then hit Enter.
 		if err != nil {
 			return fmt.Errorf("failed to read input: %s", err)
 		}
-		projectPath = projpth
+
+		projectPath = trimProjectpath(projpth)
```

Comment:
`strings.Trim` function's second parameter is a cutset, which means all leading and trailing Unicode code points contained in cutset will be removed. There is no need for loop over the items in the cutset.

```go
projectPath = strings.Trim(strings.TrimSpace(projpth), "'\"")
```

Reply:
Fixed it. 
Still, I did not remove the trimProjectpath() because with that method it is easier to make unit tests.